tutorials/analysis/vonBertalanffy/tutorial_vonBertalanffy.m

```diff
@@ -119,17 +42,26 @@
 modelFileName = [modelName '.mat']
 modelDirectory = getDistributedModelFolder(modelFileName); %Look up the folder for the distributed Models.
 modelFileName= [modelDirectory filesep modelFileName]; % Get the full path. Necessary to be sure, that the right model is loaded
-model = readCbModel(modelFileName);
+
 switch modelName
     case 'iAF1260'
+        model = readCbModel(modelFileName);
         if model.S(952, 350)==0
             model.S(952, 350)=1; % One reaction needing mass balancing in iAF1260
         end
         model.metCharges(strcmp('asntrna[c]', model.mets))=0; % One reaction needing charge balancing
         
     case 'Recon3.0model'
-        model.metFormulas{strcmp(model.mets,'h[i]')}='H';
-        model.metFormulas(cellfun('isempty',model.metFormulas)) = {'R'};
+      model = loadIdentifiedModel(modelName, modelDirectory);
```

Comment:
This function circumvents readCbModel and should only be used with models that are compliant with the model field definitions. Why not use getDistributedModel or readCbModel ?
As it currently stands, recon3 is anyways not distributed with the toolbox, so using this as default is kind of odd.

Reply:
The problem with those functions are that they return error with recon3 model. `loadIdentifiedModel` works. `readCbModel` says there are no valid models in the file, while `getDistributedModel` that the model is not present or does not belong to the distributed models.

Reply:
Is Recon3D available now, because I don't find it online yet? The thing with `loadIdentifiedModel` is that this `load`s a model, without checking the structure, and if readCbModel throws errors, than likely the model struct does violate the definitions, and should thus be fixed.  But without having the version that you use I can't say what is wrong, but its likely that we will have to adapt the model correction mechanisms for that model.